### PR TITLE
Currently running builds endpoint

### LINKED ISSUE #36: View which builds are currently running
An endpoint that shows which builds are currently running/queued would be very helpful.

### PR DESCRIPTION
**Pull request by hunterboerner**
An endpoint to show the running builds.

Fixes #36.

### CODE REVIEW

**Comment by hunterboerner** [comment]
Note for myself: steps may need to be indexed by status and builds by config_extracted

**Comment by hunterboerner** [comment]
@alanpeabody can I get a review?

File: apps/rabbitci_core/web/controllers/build_controller.ex [hunk 1]
```diff
@@ -74,4 +103,7 @@ defmodule RabbitCICore.BuildController do
         |> render("show.json")
     end
   end
+  def show(conn, %{"build_number" => bn}) when not is_integer(bn) do
```
Comment (chrismccord) [inline apps/rabbitci_core/web/controllers/build_controller.ex]: unless you pre-preprocessed the params elsewhere, `bn` will always be a string (all param keys and values are strings)

Comment (hunterboerner) [inline apps/rabbitci_core/web/controllers/build_controller.ex]: Hmm, don't remember why I added that.

**Comment by hunterboerner** [comment]
Meh. LGTM. I haven't been able to crash it (yet). Gonna merge!

**Pull request merged**